### PR TITLE
Finished AABB functions,  writed unittests #12

### PR DESCRIPTION
**Pull request by okrdima**
Finished functions Calculate,contains for FPoint2D and Fpoint3D. Writed UnitTest(AABB TEST.h)

### CODE REVIEW

File: GeometryCore/src/AABB.cpp [hunk 1]
```diff
@@ -15,17 +15,26 @@ namespace geom_utils
...
-        return{}; 
+    {
+        if (p.x <= pmax.x && p.y <= pmax.y && p.x >= pmin.x && p.y >= pmin.y)  return true;
+        else return false;
     }
 
     void AABB2D::calculate(const Polygon& poly)
     {
-
+        auto maxX = std::max_element(poly.begin(), poly.end(), [](auto a, auto b) {return b.x > a.x; }) - poly.begin();
```
Comment (VladGribinchuk) [inline GeometryCore/src/AABB.cpp:25]: What will happen if input poly is empty Polygon?

File: UnitTests/UnitTests.vcxproj.filters [hunk 2]
```diff
@@ -1,36 +1,15 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <ItemGroup>
-    <Filter Include="Source Files">
```
Comment (VladGribinchuk) [inline UnitTests/UnitTests.vcxproj.filters:4]: Why have you removed source file filters from project? Is this unitentionally?

Comment (okrdima) [reply UnitTests/UnitTests.vcxproj.filters:4]: Oy, i am sorry, yes, is unitentionally

File: GeometryCore/src/AABB.cpp (hunk 1 above)
Comment (okrdima) [reply GeometryCore/src/AABB.cpp:25]: Didn't think about it. Need to chek? Is it all?

Comment (VladGribinchuk) [reply GeometryCore/src/AABB.cpp:25]: U can check and u will see) Please fix it

**Pull request closed without merging**